src/formElementPlugins/ApplicationViewWrapper.tsx

```diff
@@ -251,7 +255,7 @@ export const buildParameters = (
   const simpleParameters: any = {}
   const parameterExpressions: any = {}
   for (const [key, value] of Object.entries(parameters)) {
-    if (value instanceof Object && !Array.isArray(value)) {
+    if (value instanceof Object && !Array.isArray(value) && 'operator' in value) {
```

Comment:
as per our convo, would prefer for a method from evaluator to deduce this (not required for now)

Reply:
Cool, will bear it in mind next time we update the evaluator.